Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldsHelper.cs

```diff
@@ -81,61 +81,61 @@ public static int CreateField(IRSAPIClient client, FieldRequest request)
 			//Set the workspace ID
```

Comment:
the whole method logic should be in a try/catch

Reply:
Done